inspirehep/modules/workflows/workflows/article.py

```diff
@@ -304,53 +308,47 @@
 
 
 SEND_TO_LEGACY_AND_WAIT = [
-    IF_ELSE(
+    send_robotupload(
+        marcxml_processor=hep2marc,
+        mode="replace"
+    ),
```

Comment:
This would not work in case of a new record because we have not yet obtained a recid from legacy (this is something that we can do but in a later step).